### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by TIDIALLO** [kinfey/MSFabricCopilotWorkshop#10]
Hi,
**_var jsonKBContent = System.Text.Json.JsonSerializer.Deserialize>(setKBContent);_**

II think something is missing on this line thanks for your reply

**Comment by TIDIALLO** [comment]
yes I loaded the ones on the doc folder in my storage account in a container via the admin interface (which I found cool with streamlit

**Comment by TIDIALLO** [comment]
kubectl get nodes
E1104 11:30:09.666568   16512 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp [::1]:8080: connect: connection refused

**Comment by diallo66** [comment]
kubectl get nodes
E1104 11:30:09.666568 16512 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp [::1]:8080: connect: connection refused

HI all, need your help for this issue
